akka-docs/rst/java/camel.rst

```diff
@@ -1,9 +1,16 @@
 
 .. _camel-java:
 
-#############
- Camel
-#############
+Camel
+#####
+
+.. warning::
+  Akka Camel is deprecated in favour of `Alpakka`_ , the Akka Streams based collection of integrations to various endpoints (including Camel).
+  Refer to the `Alpakka documentation`_ for more information.
+
+.. _Alpakka: https://github.com/akka/alpakka
+.. _Alpakka documentation: http://developer.lightbend.com/docs/alpakka/latest/
```

Comment:
`latest` is not the link we should use, because that is snapshot docs. We are missing `current` feature in TechHub so far. I think it might be enough to link to the github repo.